variables.tf

```diff
@@ -37,6 +37,12 @@ variable "config_output_path" {
   default     = "./"
 }
 
+variable "config_file_permissions" {
+  description = "File mode of the Kubectl config file. For security concerns this defaults to 0600."
+  type        = string
+  default     = "0600"
```

Comment:
I'll set default to `null` in that way we use the default permission set by the local_file resource https://www.terraform.io/docs/providers/local/r/file.html#file_permission. Furthermore we don't change the current module's behavior => avoid breaking something.